README.md

```diff
@@ -1,86 +1,10 @@
-- [nodejs & npm (LTS)](https://nodejs.org/en/download/) Node Package Manager will provide tooling for bulding our front end web site.
-
-### Recommended ###
-- [Visual Studio 2022](https://visualstudio.microsoft.com/downloads/). Version 17.9 or later. The community edition is fine, though we believe most students should have access to higher SKUs with EWU's MSDN (this assumption may be wrong). Lab computers should have Visual Studio Professional already installed. This should install .NET 8.0.
-- [GitKraken](https://gitkraken.keboo.dev/) Though you can do all of the git interaction from within Visual Studio or on the command line, GitKraken is free for open source work. It also provides a nice graphical version of the commit history so you can see how various commits and branches relate. 
-- [GitHub Student Developer Pack](https://education.github.com/students) Lots of great development tools and resources. The JetBrains products ReSharper (a plugin for Visual Studio), and Rider (a full C# IDE) are great tools for helping you write better code. 
-
-### VS Code Extensions used in class ###
-- [Vitest](https://marketplace.visualstudio.com/items?itemName=ZixuanChen.vitest-explorer) Unit testing extension
-- [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) code formatting extension
-- [Volar](https://marketplace.visualstudio.com/items?itemName=Vue.volar) Vue tooling
-- [TypeScript vue Plugin (Volar)](https://marketplace.visualstudio.com/items?itemName=Vue.vscode-typescript-vue-plugin) Vue TypeScript Tooling
-- [Code Spell Checker](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker)
-
-### Troubleshooting issues in VS Code
-- Make sure you run `npm i` or `npm ci`
-- Make sure all the extensions are installed
-- Restart VS Code
-- Pull the repo again and see if the unmodified version works
-
-### Azure Issues
-If you are having issues gettig Azure setup correctly and everything working, please check out this document: https://github.com/IntelliTect-Samples/EWU-CSCD379-2024-Spring/blob/main/SettingUpAzure.md
+1. Once upon a time, there was a hurdle, the Wordle hurdle. It was a devious hurdle, one that ___always___ wanted words. Every day it would eat millions of words, and it still wanted more. The only way to defeat it was to figure out what words it was asking for.
+2. The words that our Wordle app requires are the keys to a puzzle... a puzzle freeing the world from the eternal grasp of spelling errors. If we're able to find the correct words, everyone's spelling will be perfect, FOREVER!!!!!
+3. We could implement a treasure hunt with the wordle words. Like use it to string us along a path to something. It could be a game: figure out a word to get to the castle, figure out a word to unlock the dragon from the dungeon, figure out a word to learn to fly the dragon, etc. The word could be related to the action you're trying to do (the user would know that).
```

Comment:
I like the #3 idea a lot! Sounds fun and epic. good job attempting EC as well

Reply:
Thanks :)